core/sys/windows/hidsdi.odin

```diff
@@ -0,0 +1,41 @@
+// +build windows
+package sys_windows
```

Comment:
Why is this in a separate file to the rest when they use the same system library `hid.lib`?

Reply:
No particular reason, I was just following the header files for the SDK, wasn't sure about the implications of importing the hid library more than once.

Reply:
I'd recommend moving them to one file, rather than doing the same thing as the header files.